srcpkgs/vba-m/template

```diff
@@ -1,10 +1,9 @@
 # Template file for 'vba-m'
 pkgname=vba-m
-reverts=1292_2
-version=2.1.4
-revision=4
+version=2.1.5
+revision=1
 build_style=cmake
-configure_args="-DENABLE_GTK=TRUE -DENABLE_FFMPEG=TRUE -DENABLE_LINK=TRUE"
+configure_args="-DENABLE_GTK=TRUE -DENABLE_FFMPEG=TRUE -DENABLE_LINK=TRUE -DCMAKE_BUILD_TYPE=Release"
```

Comment:
Why is this necessary?

Reply:
Build was failing otherwise. Is there a xbps way to do it I overlooked?

Reply:
No, I just wanted to make sure it was actually required.